src/components/questions/answersList.js

```diff
@@ -8,20 +8,47 @@ class AnswerList extends React.Component{
       answers: null,
       display: 2
     };
+    this.determineDisplayed = this.determineDisplayed.bind(this);
+    this.seeMoreAnswersClick = this.seeMoreAnswersClick.bind(this);
   }
 
-  makeAnswerArray(answerObj){
+  //DETERMINES NUMBER OF ANSWERS RENDERED
+  determineDisplayed(propsAnswers){
```

Comment:
The name of this function made me think it was calculating the actual number of answers to display. Maybe something like getDisplayedAnswers or just getDisplayed would be clearer?